modules/selecting-cluster-capabilities.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// *installing/cluster-capabilities.adoc
+
+:_content-type: PROCEDURE
+[id="selecting-cluster-capabilities_{context}"]
+= Selecting cluster capabilities
+You can select cluster capabilities by following one of the installation methods that allow you to customize your cluster, such as "Installing a cluster on AWS with customizations" or "Installing a cluster on GCP with customizations". 
+
+During those installations, you create an `install-config.yaml` file that contains the configuration parameters for your cluster. You can use the following configuration parameters to select cluster capabilities:
```

Comment:
Just an idea, but WDYT of the following?
```suggestion
During a customized installation, you create an `install-config.yaml` file that contains the configuration parameters for your cluster. You can use the following configuration parameters to select cluster capabilities:
```

Reply:
I like it. thanks